### ISSUE TITLE
Support unconditional file generation

### ISSUE THREAD

**Issue by euroelessar** [planetscale/vtprotobuf#40]
In order to support usage of a protoc plugin in bazel rules one needs to ensure that such plugin does always generate a file per each input file.

Would it be fine to add a feature to force a file generation? e.g. `force` feature which would just unconditionally return `true` as the result of `GenerateFile(...)`

**Comment by vmg** [comment]
Yeah that's not a problem. Feel free to open a PR. I think a better name would be `-allow-empty`
